### PR TITLE
Rename openloco

### PR DESCRIPTION
**Pull request by petergaal**

### CODE REVIEW

**Comment by janisozaur** [comment]
The PR title had me thinking it's wissel again :stuck_out_tongue:

File: src/openloco/input.h [hunk 1]
```diff
@@ -54,47 +54,47 @@ namespace openloco::input
     };
 
     void init();
-    void init_mouse();
-    bool has_flag(input_flags value);
-    void set_flag(input_flags value);
-    void reset_flag(input_flags value);
+    void initMouse();
```
Comment (duncanspumpkin) [inline src/openloco/input.h:57]: Annoyingly this is causing a colision with my PR as I did the functions in the input folder but input.h isn't in that folder. I'll probably merge this one then adjust my PR.

File: src/openloco/input.h [hunk 2]
```diff
@@ -54,47 +54,47 @@ namespace openloco::input
...
+    ui::widget_index getPressedWidgetIndex();
 
     void updateCursorPosition();
 
-    bool is_tool_active(ui::WindowType, ui::window_number);
+    bool isToolActive(ui::WindowType, ui::window_number);
     bool toolSet(ui::window* w, int16_t widgetIndex, uint8_t tool);
-    void cancel_tool();
-    void cancel_tool(ui::WindowType, ui::window_number);
+    void cancelTool();
```
Comment (duncanspumpkin) [inline src/openloco/input.h:80]: In my PR I switched this around to toolCancel to follow toolSet

**Review by duncanspumpkin — APPROVED** [review]

File: src/openloco/input.h (hunk 2 above)
Comment (petergaal) [reply src/openloco/input.h]: Ok, I renamed it.

**Pull request merged**